src/exchange.ts

```diff
@@ -15,16 +15,21 @@ import {
 } from './types';
 import { getDisplayName, hash } from './utils';
 import { parse } from 'graphql';
+/* eslint-disable-next-line */
+// @ts-ignore
+import { version } from '../package.json';
 
 export const devtoolsExchange: Exchange = ({ client, forward }) => {
-  // Disable in prod and SSR
-  if (process.env.NODE_ENV === 'production' || typeof window === 'undefined') {
+  if (
+    typeof window === 'undefined' ||
+    process?.env?.NODE_ENV === 'production'
```

Comment:
@JoviDeCroock given the changes to the client, devtools is no longer an option in prod so we can add this check in the exchange 🌴

Reply:
Yes, we might have to add a disclaimer like:

This plugin will be automatically disabled in production if you build with
```
    new webpack.DefinePlugin({
      'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
    }),
```

This is done by default in CRA, Next, .... so it's mainly targetted at people running their own builds